src/com/github/Kraken3/AFKPGC/EventHandlers.java

```diff
@@ -46,10 +46,6 @@ public void onPlayerMoveEvent(PlayerMoveEvent event) {
 		registerActivity(event.getPlayer());
 	}	
 	@EventHandler
-	public void onPlayerChatEvent(PlayerChatEvent event) {

```

Comment:
Yes, hence my questioning of it. I suppose it makes sense in the broader reasoning being applied -- you might be using a simple AFK machine in the largest lag machine on the server, and happily chatting with your friends. That doesn't mean you shouldn't be kicked.